src/doc/trpl/unsafe.md

```diff
@@ -49,32 +49,15 @@ and mutable (`&mut`) references have some aliasing and freezing
 guarantees, required for memory safety.
 
 In particular, if you have an `&T` reference, then the `T` must not be
-modified through that reference or any other reference. There are some
-standard library types, e.g. `Cell` and `RefCell`, that provide inner
-mutability by replacing compile time guarantees with dynamic checks at
-runtime.
+modified through that reference or any other reference or pointer and must not
+alias with any `&mut` reference. There are some standard library types, e.g.
```

Comment:
I can adjust the text to use the term "reference" for `&T/&mut T`, "raw pointer" for `*const T/*mut T` and "pointer" for both references and raw pointers. Would it be ok?